jwst/extract_1d/extract.py

```diff
@@ -3274,9 +3273,9 @@ def populate_time_keywords(input_model, output_model):
             spec = output_model.spec[n]             # n is incremented below
             spec.int_num = int_num[row]
             spec.time_scale = "UTC"
-            spec.start_time_mjd = start_utc[row]
-            spec.mid_time_mjd = mid_utc[row]
-            spec.end_time_mjd = end_utc[row]
+            spec.start_utc = start_utc[row]
```

Comment:
#4602 changed the meta attribute names "start_utc", "mid_utc", and "end_utc" to "start_time_mjd", "mid_time_mjd", and "end_time_mjd" in order to make them consistent with the names for these keywords that are now used in the core schema. This change is reverting that update. The new names are used in the multispec schema and hence the code here also needs to use the new names (the old ones aren't defined in the schema anymore).

Reply:
Thanks; I'll fix this.